projects/controllable_dialogue/controllable_seq2seq/controllable_seq2seq.py

```diff
@@ -836,6 +835,10 @@ def beam_search(
         beams : list of Beam instances defined in Beam class, can be used for any
                 following postprocessing, e.g. dot logging.
         """
+        if wd_features is None:
+            wd_features = []
+        if wd_wts is None:
+            wd_wts = []
```

Comment:
@stephenroller fyi, lint_changed caught this

Reply:
Yeah this was filed as #1891.